dragonfly_schema/cli/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+Command Line Interface (CLI) entry point for dragonfly-schema.
+
+Note:
+    Do not import this module in your code directly. For running the commands,
+    execute them from the command line or as a subprocess (e.g.
+    ``subprocess.call(['dragonfly-schema', 'viz'])``)
+
+Honeybee is using click (https://click.palletsprojects.com/en/7.x/) for creating the CLI.
+"""
+
+try:
+    import click
+except ImportError:
+    raise ImportError('click module is not installed.')
+
+import sys
+import os
+import logging
+import json
+
+from dragonfly_schema.model import Model
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for validating JSON files.')
+def main():
+    pass
+
+
+@main.command('viz')
+def viz():
+    """Check if dragonfly-schema is flying!"""
+    click.echo('viiiiiiiiiiiiizzzzzzzzz!')
+
+
+@main.command('validate-model')
```

Comment:
I suggest to create a group of commands for `validate` and then add model as a sub-command. That will be easier to extend if one wants to add a new command to validate a face or some other object.